plugins/Goals/API.php

```diff
@@ -260,22 +265,34 @@ private function checkDescription($description)
         return urldecode($description);
     }
 
-    private function checkPatternType($patternType)
+    private function checkPatternType($patternType, $matchAttribute)
     {
         if (empty($patternType)) {
             return '';
         }
 
         $patternType = strtolower($patternType);
 
-        $validator = new WhitelistedValue(['exact', 'contains', 'regex']);
+        if (in_array($matchAttribute, GoalManager::$NUMERIC_MATCH_ATTRIBUTES)) {
+            $validValues = ['>'];
```

Comment:
not sure, should we maybe use `greater_than` or so?

Reply:
I think we do this in some other places.

Reply:
There's a limit on the field size, I think 'greater_than' was a little too long. Didn't want to create a db upgrade for something so small, but I can if it's better.

Reply:
it's fine to have an update for it as it's not a log table.